modules/katamari/src/main/ts/ephox/katamari/api/Type.ts

```diff
@@ -23,7 +23,7 @@ const eq = <T> (t: T) => (a: any): a is T =>
 export const isString: (value: any) => value is string =
   isType('string');
 
-export const isObject: (value: any) => boolean =
+export const isObject: (value: any) => value is Object =
```

Comment:
Tested Object vs object vs Record and Lee did some research and this seems correct?